.vscode/settings.json

```diff
@@ -5,4 +5,5 @@
     "importSorter.importStringConfiguration.spacingPerImportExpression.afterStartingBracket": 0,
     "importSorter.importStringConfiguration.spacingPerImportExpression.beforeEndingBracket": 0,
     "typescript.format.enable": false,
+    "prettier.trailingComma": "none",
```

Comment:
Hmm I'd like to get to the bottom of this rather than just turning trailingComma off. When I save AddHoldingForm it just puts the destructured props one on each line? So it's no problem? The trailing comma was only a problem when they were all on one line, sorry if that wasn't clear.